background_scripts/completion.js

```diff
@@ -489,6 +491,7 @@ class TabCompleter {
           queryTerms,
           description: "tab",
           url: tab.url,
+          favIconUrl: tab.favIconUrl,
```

Comment:
It seems we're passing this along because tab.favIconUrl is sometimes a higher quality image than what can be fetched from chrome's favicon API? Is so, we should mention that here in a comment, with a reference. [This](https://issues.chromium.org/issues/40856652) is the Chromium bug I found about it, and some [related discussion](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/qS1rVpQVl8o/m/qmg1M13wBAAJ?pli=1).